e2e/cypress/integration/onboarding/cloud_sysadmin_onboarding_spec.js

```diff
@@ -0,0 +1,421 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import {testWithConfig} from '../../support/hooks';
+
+const adminSteps = ['complete_profile', 'team_setup', 'invite_members', 'hide'];
+
+describe('Cloud Onboarding - Sysadmin', () => {
+    let townSquarePage;
+    let sysadmin;
+
+    testWithConfig({
+        ServiceSettings: {
+            ExperimentalChannelSidebarOrganization: 'default_on',
+        },
+    });
+
+    before(() => {
+        // # Check if with license and has matching database
+        cy.apiRequireLicenseForFeature('Cloud');
+        cy.requireServerDBToMatch();
+
+        cy.apiInitSetup().then(({team}) => {
+            townSquarePage = `/${team.name}/channels/town-square`;
+        });
+
+        cy.apiAdminLogin().then((res) => {
+            sysadmin = res.user;
+        });
+    });
+
+    beforeEach(() => {
+        // # Login as sysadmin and set all steps to false
+        const preference = {
+            user_id: sysadmin.id,
+            category: 'recommended_next_steps',
+            value: 'false',
+        };
+
+        cy.apiSaveUserPreference(adminSteps.map((step) => ({...preference, name: step})));
+        cy.visit(townSquarePage);
+    });
+
+    /*
+     *  Happy Path
+     */
+
+    it('Cloud Onboarding - Full Flow', () => {
+        // * Make sure channel view has loaded
+        cy.url().should('include', townSquarePage);
+
+        // * Check to make sure card is expanded
+        cy.get('.Card__body.expanded .CompleteProfileStep').should('be.visible');
```

Comment:
Not sure what you mean by this, did you want me to query it a different way? I don't think there's anything wrong with using the CSS selector, since there won't be any other elements with this particular path.

Reply:
I agree. It's only something to improve for testing. Use of selector like this is [susceptible to change](https://docs.cypress.io/guides/references/best-practices.html#How-It-Works) and closer to implementation details rather than how the user actually sees it.
One approach might be to verify content which could represent that the body is expanded, like
```javascript
cy.findByText('Enter your name').should('be.visible');
```